README.md

```diff
@@ -291,7 +291,9 @@ The reset button makes it possible to test crash handling code. Press the
 button to simulate a crash. Your algorithm should periodically check if the
 button was pressed via `wasReset`. If so, your algorithm should reset any
 internal state and then call `ackReset` to send the robot back to the beginning
-of the maze.
+of the maze. After a crash, the commands `moveForward`, `turnRight`, and
+`turnLeft` will be ignored (the simulator will return "crash" and the mouse will
+not move) until `ackReset` is called.
```

Comment:
We should differentiate a simulated crash (e.g., reset button press) from an algorithm-aware crash. Unfortunately, the current wording in this section is pretty confusing. How about something like this:

> The reset button makes it possible to test code for resetting the mouse, e.g., after a malfunction. Press the button to indicate that the mouse needs to be reset. Your algorithm ... to the beginning of the maze.

> Related but different: if the algorithm attempts to drive into a wall and receives a "crash" response from the simulator, all subsequent movement commands will do nothing and will return "crash" until `ackReset` is called.

> To summarize: the reset button simulates a hypothetical crash. Use it to test how your mouse behaves when it detects a crash. If the mouse actually crashes within the simulation, you must reset it before it can move again.

It's probably worth *also* updating the "Mouse API" section to include some information about the new behavior.